backend/logic/printful/webhook.js

```diff
@@ -207,18 +224,17 @@ const processShippedEvent = async (event, shopId) => {
  * Processes a ProductUpdated event sent by Printful via webhook.
  * @param event
  * @param shopId
+ * @param shopData
```

Comment:
nit: document the type for shopData (and the other params while we are at it).

Reply:
Okay, I will update

Reply:
Updated